backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/impl/ElasticsearchSearchIndexesContext.java

```diff
@@ -29,4 +30,8 @@
 
 	ElasticsearchSearchFieldContext field(String absoluteFieldPath);
 
-}
+	ElasticsearchIndexSchemaNamedPredicateNode namedPredicate(String absoluteNamedPredicatePath);
+
+	boolean hasSchemaObjectNodeComponent(String absoluteFieldPath);
```

Comment:
It looks like a git conflict resolution went wrong. This has been removed in master, please remove it here.